self-hosting-novu/s3.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: 'S3'
+description: 'Using S3 as a storage backend for your application'
+icon: 'text-height'
+---
+
+For local development we use [LocalStack.cloud](https://localstack.cloud/) is utilized as a local emulation of S3 during development.
```

Comment:
For local development we use [LocalStack.cloud](https://localstack.cloud/) is utilized as a local emulation of S3 during development. => For local development we use [LocalStack.cloud](https://localstack.cloud/). It is utilized as a local emulation of S3 during development.